command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
On Thu, Jul 28, 2016 at 02:56:57PM -0700, W. Trevor King wrote:
> On Thu, Jul 28, 2016 at 02:49:18PM -0700, Mike Brown wrote:
> > Ok... I must have missed the memo where we decided to go back on the
> > premise of supporting other Ux choices…
> 
> We are not [1].

And I think the language in [1,2] makes this clear.  I'll push
something along those lines into this PR once I have a link to a
compliance-project homepage (I've just asked for one on Slack's
#general, since that's where I see the most trademark chatter).

[1]: https://github.com/opencontainers/runtime-spec/pull/513/files#r72701237
[2]: https://github.com/opencontainers/runtime-spec/pull/513/files#r72703237